oas_docs/output/kibana.serverless.yaml

```diff
@@ -344,6 +345,21 @@ paths:
                 name:
                   description: The display name for the connector.
                   type: string
+              required:
+                - name
+                - connector_type_id
+          application/json; Elastic-Api-Version=2023-10-31:
```

Comment:
`Elastic-Api-Version=2023-10-31` here is added via overlays in `<kibana_root>/oas_docs/overlays`. Is it planned to be addressed in a follow up PR?